Source/WebGPU/WebGPU.xcodeproj/xcshareddata/xcschemes/WebGPU.xcscheme

```diff
@@ -20,6 +20,20 @@
                ReferencedContainer = "container:WebGPU.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "97FA1A8F29C086CE0052D650"
+               BuildableName = "All"
+               BlueprintName = "All"
+               ReferencedContainer = "container:WebGPU.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
I don't think this is supposed to be part of the commit

Reply:
Why? I created an `All` target to match what we do in other projects, but I can remove it.

Reply:
oh ok, keeping sounds fine then